docs/ports.md

```diff
@@ -0,0 +1,13 @@
+OpenShift Reserved Ports
+========================
+
+OpenShift reserves all ports on hosts that are reserved by [upstream Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/master/ports/ports.go) as well as the following host ports:
+
+| 2379,2380 | masters | [etcd](https://github.com/openshift/installer) | etcd API |
+| 6443 | masters |[kube-apiserver](https://github.com/openshift/cluster-kube-apiserver-operator) | Kubernetes API |
+| 9099 | masters | [Cluster Version Operator](https://github.com/openshift/cluster-version-operator) | Metrics |
+| 9100, 9101 | nodes | [Node Exporter](https://github.com/openshift/node_exporter) | Metrics |
```

Comment:
Does "nodes" mean "workers" (a.k.a. "compute", etc.), or does it mean "all machines in the cluster, regardless of role"?

Reply:
All machines in the cluster.  This could be more formal (as the mentioned e2e test)